contracts/Fees.sol

```diff
@@ -61,6 +65,13 @@ abstract contract Fees is IFees, Owned {
         }
     }
 
+    function _fetchDynamicFee(PoolKey memory key) internal view returns (uint24 dynamicFee) {
```

Comment:
nit: specify as swapFee? ie `_fetchDynamicSwapFee` and return `dynamicSwapFee` for clarity given we also have withdrawal fees?